### DISCUSSION TITLE
请教在h5页面怎么完成截图动作并在服务器上留存

### DISCUSSION THREAD

**Discussion by BrtMeister** [ly525/luban-h5#220] [Q&A]
想在h5页面中完成表格填充之后，做一个截图发送到服务器的一个动作，求大佬赐教

**Comment by ly525** [comment]
> 想在h5页面中完成表格填充之后，做一个截图发送到服务器的一个动作，求大佬赐教

你意思是保存一个H5的封面图在服务器？

**Reply by ly525** [reply]
![image](https://user-images.githubusercontent.com/12668546/92002192-27eb2d00-ed72-11ea-8293-e402a8ed8bca.png)

**Reply by ly525** [reply]
![image](https://user-images.githubusercontent.com/12668546/92003090-2ff79c80-ed73-11ea-9a08-5b4d78e82e96.png)

![image](https://user-images.githubusercontent.com/12668546/92003146-430a6c80-ed73-11ea-8c3c-b82b7afa9902.png)

**Reply by BrtMeister** [reply]
就是比如说我的h5页面是一个需要填写表单的互动页面，我想当别人填写完之后有一个保存的按钮，然后做一个动作截图到我本地服务器

**Comment by BrtMeister** [comment]
就是比如说我的h5页面是一个需要填写表单的互动页面，我想当别人填写完之后有一个保存的按钮，然后做一个动作截图到我本地服务器

**Reply by ly525** [reply]
目前鲁班的功能 更多是偏向于通用型功能，这个有点特殊定制化了。
我个人偏向于，或者建议 可以把收集到的用户数据，然后起一个海报生成服务，来生成对应的海报，来满足的你的诉求。
